BUILD

```diff
@@ -78,7 +79,15 @@ java_deps(
     name = "console-deps",
     target = ":console-binary",
     java_deps_root = "console/services/lib/",
-    visibility = ["//visibility:public"]
+    visibility = ["//visibility:public"],
+)
+
+deploy_distribution(
```

Comment:
What if we just deploy the complete distribution (as produced by `//:assemble-linux-targz`, as opposed to`//:console-deps`?

I think the developers would appreciate working with a file format that's more common. The output of `//:console-deps` feels like we're working with "intermediary format".